package.json

```diff
@@ -8,6 +8,7 @@
     "lint": "eslint --ext js,jsx src",
     "test": "jest src",
     "test:cover": "jest src --coverage",
+    "test:watch": "jest src -- --watch",
```

Comment:
Unnecessary `--`, there can be one of the two options:
* `npm test -- --watch`
* `jest src --watch`

Reply:
fixed